pkg/vault/contracts/Vault.sol

```diff
@@ -344,7 +344,6 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
      */
     struct SwapInternalLocals {
         uint256 totalSwapFeeAmountScaled18;
-        uint256 totalSwapFeeAmountRaw;
```

Comment:
I tried it - we can get rid of this struct now without blowing up the stack. Can define `totalSwapFeeAmountScaled18` on top, and `aggregateFeeAmountRaw` down where it's first used.